### DISCUSSION TITLE
脚本部署安装问题

### DISCUSSION THREAD

**Discussion by szsyzx** [zdz/ServerStatus-Rust#161] [General]
再次提问希望老大不要生气，虽然您的文档写的很清楚，但是对于小白我来说，还是有的不理解。希望你能耐心解答下

1.#服务端安装：
wget --no-check-certificate -qO status.sh 'https://raw.githubusercontent.com/zdz/ServerStatus-Rust/master/status.sh'
bash status.sh -i -s【服务器IP为X.X.X.X】
2#安装 客户端
wget --no-check-certificate -qO status.sh 'https://raw.githubusercontent.com/zdz/ServerStatus-Rust/master/status.sh'
bash status.sh -i -c
[注意] 请输入服务端的信息, 格式为 "protocol://username:password@master:port"
protocol://h1:p1@X.X.X.X:8080

不知道这样的输入是否正确。

这里的username对应以下的name ,password 对应以下的password  master对应服务段IP port要么是 9394 或者8080
grpc_addr = "0.0.0.0:9394"
http_addr = "0.0.0.0:8080"

hosts = [
  {name = "h1", password = "p1", alias = "n1", location = "🏠", type = "kvm", labels = "os=arch;ndd=2022/11/25;spec=2C/4G/60G;"},
  {name = "h2", password = "p2", alias = "n2", location = "🏢", type = "kvm", disabled = false},
  {name = "h3", password = "p3", alias = "n3", location = "🏡", type = "kvm", monthstart = 1},
  {name = "h4", password = "p4", alias = "n4", location = "cn", type = "kvm", notify = true, labels = "ndd=2022/11/25;spec=2C/4G/60G;"},

  # 最小化配置
  {name = "mac", password = "pp", alias = "macos"},
  {name = "pi", password = "pp", alias = "pi", labels = "os=pi"},
  {name = "win", password = "pp", alias = "windows"},
  {name = "android", password = "pp", alias = "android", labels = "os=android"},
]

这里的

# eg:【这里eg什么意思啊】
bash status.sh -i -c grpc://h1:p1@127.0.0.1:9394
bash status.sh -i -c http://h1:p1@127.0.0.1:8080

127.0.0.1这个是服务端和客户端在同一台服务器上安装的方式吗

如果服务器和客户端不是同一个机器这里的127.0.0.1是不是需要改成服务端的IP地址

**Comment by zdz** [comment]
https://github.com/zdz/ServerStatus-Rust/issues/2

**Comment by zdz** [comment]
同一个机器 127.0.0.1 可以不用修改
